frontend/src/components/commons/SideNavigation.jsx

```diff
@@ -1,18 +1,82 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
 
+const styles = {
+  buttonList: {
+    width: 200,
+    paddingRight: 25,
+    float: "left"
```

Comment:
You should be able to use "flex" for this too since you did for the other element.

Reply:
fixed in 539ddb3